Gifski/MainWindowController.swift

```diff
@@ -301,6 +305,14 @@ final class MainWindowController: NSWindowController {
 		}
 	}
 
+	@IBAction private func quickLook(_ sender: Any) {
```

Comment:
Instead of having this here to merely forward the call. I was thinking we instead only have `func quickLook(_ sender: Any) {` in `ConversionCompletedView` and somehow inject `ConversionCompletedView` into the responder chain so `ConversionCompletedView` can respond directly to the `quickLook` message. That way we wouldn't need https://github.com/sindresorhus/gifski-app/pull/79/files#diff-53ab4064437ff40d094178968ddb4d1fR335 either. (You might have to change the view to be removed/added instead of shown/hidden so it doesn't respond when it's not visible. I'm not sure whether it would respond if it's hidden or if there's anyway to prevent that).

Reply:
Yeah, the problem with responder chain is currently beyond me. I think we would need a controller, but I don't have the knowledge needed to make the connection between the window and conversion view in a responder chain. I would use some help here.

Reply:
Alright, spent the day on learning how does exactly responder chain work in AppKit and fixed the problem with a proper solution.

Reply:
@sunshinejr Awesome! This is exactly what I conceptually had in mind, but I didn't know how to do it in practice either.

Any recommended reading material?

Reply:
@sindresorhus I had lots of problems as some of the methods of the responder chain in AppKit are different than the ones in UIKit (e.g. `becomeFirstResponder()` is quite the opposite, lol). But yeah basically reading the docs of responder chain on apple doc archive ([this](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/EventOverview/EventArchitecture/EventArchitecture.html) one for instance). 

I also played with all the properties to make sure I understand the differences (e.g. `initialFirstResponder` vs `firstResponder` etc). 

I have lots of material that I want to convert into a blog post as I didn't found one that goes into what I experienced while implementing this one.